modules/auxiliary/scanner/mssql/mssql_login.rb

```diff
@@ -30,12 +30,21 @@ def initialize
           'BLANK_PASSWORDS' => true
         }
     )
+    register_options(
+      [
+        Opt::RHOST,
+        Opt::RPORT(1433),
+        OptString.new('USERNAME', [ false, 'The username to authenticate as', 'sa']),
+        OptString.new('PASSWORD', [ false, 'The password for the specified username', '']),
+        OptBool.new('TDSENCRYPTION', [ true, 'Use TLS/SSL for TDS data "Force Encryption"', false]),
+        OptBool.new('USE_WINDOWS_AUTHENT', [ true, 'Use windows authentication (requires DOMAIN option set)', false]),
+      ])
```

Comment:
This change doesn't seem right, shouldn't there be a mixin pulling these in (or at least the common ones like RHOST and RPORT) for the mssql modules?
Similar to the [smb_login](https://github.com/rapid7/metasploit-framework/blob/1abaef494566c99a122d56247fb3ed26d252ae87/modules/auxiliary/scanner/smb/smb_login.rb#L11) module here still includes a mixin registering these options but makes use of an SMB client too

Reply:
this module runs differently since it's coming from the login_scanner rather than the client directly, but I'll investigate pulling this out, that's definitely the cleaner approach